status-im/status-react
# [15569] Select recent tab on messages home when double tapping messages icon

fixes #15569 

### Summary
This PR uses `GestureHandler` to select the recent tab on the messages home screen, This was achievable by moving the selected tab `atom` to `app-db` and not to be local state inside the UI component to make it accessible from anywhere.

### Review notes
I didn't write tests for the event because it was really simply, Just `assoc`ing a keyword.

### Testing notes
Double tap on the bottom navigation's messages icon and see if recent tab is selected.

#### Platforms


- Android
- iOS

### Steps to test

- Open Status
- Navigate to the messages tab using the bottom nav
- Select any tab but recent, Like contacts or groups
- Double tap on the messages tab on the bottom navigation, And see if recent tab is selected or not

status: ready
